### ISSUE TITLE
Max results query parametr

### ISSUE THREAD

**Issue by smythp** [JonathanReeve/corpus-db#21]
Would be nice to have a query parameter that can return x number of results to speed up examples in the notebook.

**Comment by JonathanReeve** [comment]
Good idea. Yeah `/api/subjects` takes forever, and I don't know how to do profiling in Haskell, yet. I was thinking of caching stuff like that, since it's always going to be the same (for the moment, at least).

**Comment by smythp** [comment]
Caching sounds like a good idea. Can you support query parameters yet? You'll need them to do stuff likein the fancy SQL queries you showed off in your presentation.

**Comment by JonathanReeve** [comment]
Scotty has a parameter parser built-in, I'm pretty sure. [The tutorial here](http://adit.io/posts/2013-04-15-making-a-website-with-haskell.html) says to do something like this: 

```haskell
get "/hello" $ do
  name <- param "name"
  text name
```

Which I imagine handles a URL like `/hello?name="Jonathan"`, although I haven't tested it.
